backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/SubscriptionCreateModel.java

```diff
@@ -8,18 +8,21 @@
 import ca.gov.dtsstn.cdcp.api.web.validation.LanguageCode;
 import ca.gov.dtsstn.cdcp.api.web.validation.LanguageCodeValidator.CodeType;
 import io.swagger.v3.oas.annotations.media.Schema;
-import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.NotBlank;
 
 @Immutable
 @Schema(name = "SubscriptionCreate")
 @JsonDeserialize(as = ImmutableSubscriptionCreateModel.class)
 public interface SubscriptionCreateModel {
 
-	@NotNull
+	
+	@NotBlank
 	@Schema(example = "CDCP")
 	@AlertTypeCode(message = "Alert type code does not exist")
 	String getAlertTypeCode();
 
+	
+	@NotBlank
```

Comment:
**Massive Nitpick:** We can remove the newly introduced extra lines before the `@NotBlank` annotation:

```suggestion
	@NotBlank
	@Schema(example = "CDCP")
	@AlertTypeCode(message = "Alert type code does not exist")
	String getAlertTypeCode();

	@NotBlank
```

Reply:
done